### PR TITLE
Update monitor squads

### PR DESCRIPTION
**Pull request by diegopetrucci**
The monitor squad has recently been split into three squadlets, so the playbook should reflect that.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,14 +66,15 @@ By definition, members work on their respective squad, although they are free to
...
 | Dev Experience                | Ilya, Olivier, David                        |    3/3       | [#mobile-tooling](https://babylonhealth.slack.com/messages/CL3QSC2NM) |
 | Early Feedback                | Danilo                                      |    1/1       | [#early_feedback](https://babylonhealth.slack.com/archives/CLGA76GG7) |
 | Enrolment and Integrity       | James                                |    1/1       | [#ei-squad](https://babylonhealth.slack.com/archives/CGR4D5NKX) |
 | Healthcheck                   | Ben, Julien                                 |    2/3       | [#healthcheck_tribe](https://babylonhealth.slack.com/archives/C7995CX3R) |
-| Monitor                       | Anders, Joshua, Daniel H.                      |    3/3       | [#patient-metrics-team](https://babylonhealth.slack.com/archives/CE37S5W9Z) |
+| Monitor (Bulbasaur)           | Joshua, Dan                                 |    2/2       | |
+| Monitor (Charmander)          | Anders                                      |    1/1       |  |
```
Comment (Daniel1of1) [inline README.md]: Should charmander availability be 1/2 ?

```suggestion
| Monitor (Charmander)          | Anders                                      |    1/2       |  |
```

Comment (diegopetrucci) [reply README.md]: 👀

**Review by Daniel1of1 — APPROVED** [review]
👍

**Review by danspndl — APPROVED** [review]

**Comment by AnnKatF** [comment]
Please always provide a description to every PR, even if it's a small one.

**Comment by AnnKatF** [comment]
Also, this PR is missing labels.

**Review by AnnKatF — DISMISSED** [review]
Please add a description and labels to the PR.
We should never have PRs opened without those.

**Comment by diegopetrucci** [comment]
@AnnKatF done

File: README.md [hunk 2]
```diff
@@ -66,14 +66,15 @@ By definition, members work on their respective squad, although they are free to
 | Squad Name                    | Members                                     | Availability | Public slack channel |
 |-------------------------------|---------------------------------------------| ------------ | -------------------- |
 | Booking                       | Witold                                      |    1/1       | |
-| Condition Management          | Catarina, João, Dan S., Diego                      |    4/4       | [#health-mgmt-public](https://babylonhealth.slack.com/archives/CCNHJUXLH) |
```
Comment (AliSoftware) [inline README.md]: Where did @AnnKatF and @NSMyself end up, now that they're not in the Condition Managment squad anymore but have not migrated to Monitor either like Dan and @diegopetrucci did?

**Comment by AliSoftware** [comment]
While we're at it. since this PR changes the Squad @Daniel1of1 is assigned to, I see that he haven't added his name in the first table. So he's "referenced" here in the Squad table but not listed at part of our team in the first table 😅

File: README.md (hunk 2 above)
Comment (diegopetrucci) [reply README.md]: You need to ask them because I honestly don't know.

Maybe @AnnKatF can shed some light.

**Comment by diegopetrucci** [comment]
> While we're at it, and since this PR changes the Squad @Daniel1of1 is assigned to: I just saw that he hasn't added his name in the first table. So he's "referenced" here in the Squad table but not listed at part of our team in the first table 😅

@Daniel1of1 could you please do this?

**Comment by Daniel1of1** [comment]
@diegopetrucci @AliSoftware done -> #224 😄

**Pull request merged**